size.txt

```diff
@@ -1,2 +1,2 @@
-Sketch uses 77946 bytes (30%) of program storage space. Maximum is 253952 bytes.
+Sketch uses 77988 bytes (30%) of program storage space. Maximum is 253952 bytes.
```

Comment:
We don't really care about the change in the size of the program, only about the change in global variables. So including this line is unnecessary and adds confusion. Please modify `scripts/testAndBuild.sh` so that line 6 reads as follows:

```
tail -n 3 output.txt | head -n 1 > size.txt
```

Then submit that file and a new (modified) `size.txt`. Otherwise, this looks good.

Reply:
Changes implemented.

Reply:
Hmm... I expected a new size file with only one line...

Reply:
So did I. For some reason the new size.txt wasn't added to my commit. Doing that now.